apps/design/frontend/src/rich_text_editor.tsx

```diff
@@ -89,6 +95,29 @@ function ControlButton({
   );
 }
 
+const PDF_PIXELS_PER_INCH = 96;
+const LETTER_PAGE_HEIGHT_INCHES = 11;
+const LETTER_PAGE_WIDTH_INCHES = 8.5;
+
+/**
+ * These assume a worst-case of ballots with full-width contest boxes and images
+ * that take up all the available width.
+ *
+ * The subtracted margins are inexact and can be tweaked if needed. Last tested
+ * with "full-height" and "full-width" images rendering without error on
+ * letter-sized pages.
+ *
+ * [TODO] The max width should really be the VxMark screen size (1080px), since
+ * that's the largest render surface for these images, but we need to add image
+ * scaling to the ballot renderer first, to limit them to the bounds of the
+ * contest box.
+ */
+const NORMALIZE_PARAMS: Readonly<images.NormalizeParams> = {
+  maxHeightPx: (LETTER_PAGE_HEIGHT_INCHES - 3) * PDF_PIXELS_PER_INCH,
+  maxWidthPx: (LETTER_PAGE_WIDTH_INCHES - 2) * PDF_PIXELS_PER_INCH,
+  minWidthPx: 1 * PDF_PIXELS_PER_INCH,
```

Comment:
Why min width but no min height here?

Reply:
Didn't think either one is really needed here, so still on the fence about whether a min width is even needed (since the images are rendered in the rich text editor as 1:1 size anyway). Will remove for now, but LMK if you have a preference for whether we set minimums or not

Reply:
I think it's probably not necessary, especially since ballots will be proofed